WhaleSpotting/Program.cs

```diff
@@ -34,6 +37,51 @@
 
 var app = builder.Build();
 
+//populate db with sample data if empty
+using var scope = app.Services.CreateScope();
+var services = scope.ServiceProvider;
+var context = services.GetRequiredService<WhaleSpottingDbContext>();
+context.Database.EnsureCreated();
+//Create Sample Data in Database
+context.Database.ExecuteSqlRaw("TRUNCATE public.\"Likes\" RESTART IDENTITY CASCADE");
+context.Database.ExecuteSqlRaw("TRUNCATE public.\"WhaleSightings\" RESTART IDENTITY CASCADE");
+context.Database.ExecuteSqlRaw("TRUNCATE public.\"WhaleSpecies\" RESTART IDENTITY CASCADE");
+context.Database.ExecuteSqlRaw("TRUNCATE public.\"Users\" RESTART IDENTITY CASCADE");
+context.SaveChanges();
+
+var species = SampleSpecies.GetSpecies();
+context.WhaleSpecies.AddRange(species);
+context.SaveChanges();
+var users = SampleUsers.GetUsers();
+context.Users.AddRange(users);
+context.SaveChanges();
+var sightings = SampleSightings.GetSightings();
+var collectionOfSightings = new List<WhaleSighting>();
+var newWhaleSpecies = new WhaleSpecies();
+var newUser = new User();
+var newWhaleSighting = new WhaleSighting();
```

Comment:
I don't think we need to declare these 3 variables here

Reply:
Removed variables (created within the foreach instead